Source/WebKit/WPEPlatform/wpe/WPEView.cpp

```diff
@@ -178,13 +180,25 @@ static void wpeViewDispose(GObject* object)
     G_OBJECT_CLASS(wpe_view_parent_class)->dispose(object);
 }
 
+static void wpeViewSetGestureController(WPEView* view, WPEGestureController* controller)
+{
+    view->priv->gestureController = adoptGRef(controller);
+}
+
+static WPEGestureController* wpeViewGetGestureController(WPEView* view)
+{
+    return view->priv->gestureController.get();
+}
+
 static void wpe_view_class_init(WPEViewClass* viewClass)
 {
     GObjectClass* objectClass = G_OBJECT_CLASS(viewClass);
     objectClass->set_property = wpeViewSetProperty;
     objectClass->get_property = wpeViewGetProperty;
     objectClass->constructed = wpeViewConstructed;
     objectClass->dispose = wpeViewDispose;
+    viewClass->set_gesture_controller = wpeViewSetGestureController;
+    viewClass->get_gesture_controller = wpeViewGetGestureController;
```

Comment:
We don't want these to be vfuncs, since the gesture controller doesn't depend on the platform implementation.